Admin/AbstractAdmin.php

```diff
@@ -422,8 +422,6 @@
      */
     protected $securityInformation = array();
 
-    protected $cacheIsGranted = array();
```

Comment:
Oh I see… well it might. But then again, every single change you do can be considered a BC-break if you think hard enough. I think in this case it's ok to dismiss this one.